com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -1365,7 +1377,23 @@ private void ConfigureSimulator()
 #endif
                 heartbeatTimeoutMS: transport.m_HeartbeatTimeoutMS);
 
+#if UTP_TRANSPORT_2_0_ABOVE
+            if (m_UseWebSockets)
+            {
+                driver = NetworkDriver.Create(new WebSocketNetworkInterface(), m_NetworkSettings);
+            }
+            else
+            {
+#if UNITY_WEBGL
+                Debug.LogWarning("WebSockets were used even though they're not selected in NetworkManager. You should adjust the setting, to silence this warning.");
```

Comment:
Would be nice to specify which setting is referred to, e.g. "you should check the 'Use Web Sockets' checkbox to silence this warning".